app/controllers/hosts_controller.rb

```diff
@@ -543,31 +543,28 @@ def submit_multiple_build
         Foreman::Logging.exception(message, error)
         success = false
       end
-      success
+      !success
     end
```

Comment:
The select block can be cleaned a bit now, the last line looks weird. I'd rename it to pick or similar, or ideally remove whole variable completely and use just true and false in both possible code flows as the begin block returns the last evaluation.